src/routes/plans/[id]/+page.svelte

```diff
@@ -289,12 +293,18 @@
   }
 
   $: if ($plan) {
+    externalDatasetNamesAbortController?.abort();
+    externalDatasetNamesAbortController = new AbortController();
+    effects
+      .getExternalDatasetNames($plan.id, data.user, externalDatasetNamesAbortController.signal)
```

Comment:
Thoughts on names vs name + schema like we do with `resourceTypes`? Could be good to mirror the latter?

Reply:
On second thought it looks like `resourceTypes` is really just reflecting that specific table. In this case it looks like `type` would be the equivalent to `schema` (from resource_types)?

Reply:
Yeah, the `type` is just the `schema`.

Reply:
Cool. At some point in the future we might have a panel dedicated to showing the available resources and maybe at that point we'd refactor this to add in `type` but for now I guess we don't have any reason to need it.